### DISCUSSION TITLE
TCStat WIND job output

### DISCUSSION THREAD

**Discussion by OliviaOstwald-NOAA** [dtcenter/METplus#2251] [Existing Builds] [answered]
I am using the WIND job arg in TCStat and my output is the ABS( ) value (i.e. ABS(AAL_WIND_34-BAL_WIND_34)). I need to generate the absolute error and bias and seem to only have absolute error. Any ideas on how to proceed? Thanks!

**Comment by JohnHalleyGotway** [comment, marked as answer]
@OliviaOstwald-NOAA, I'm guessing you've already seen this, but here's a link to the [MET User's Guide Section](https://met.readthedocs.io/en/latest/Users_Guide/tc-stat.html?highlight=TC-Stat#summary-statistics-for-columns) that describes the WIND option you're using in TC-Stat. Running with shortcut option `-column WIND` produces:
```
WIND: all wind radii errors (34-, 50-, and 64-kt) for each quadrant
```
And, as you've found, `WIND` writes out the absolute error of the wind speed differences for each of the wind radii and quadrants. However, `WIND` is just a shortcut. You can explicitly request whatever columns, differences of columns, or absolute differences of columns that you'd like. For example, if you need bias and absolute error for all 4 quadrants of the 34-kt wind, you'd request:
```
-column 'ANE_WIND_34-BNE_WIND_34,ANW_WIND_34-BNW_WIND_34,ASE_WIND_34-BSE_WIND_34,ASW_WIND_34-BSW_WIND_34'
-column 'ABS(ANE_WIND_34-BNE_WIND_34),ABS(ANW_WIND_34-BNW_WIND_34),ABS(ASE_WIND_34-BSE_WIND_34),ABS(ASW_WIND_34-BSW_WIND_34)'
```
Or if you want "all" winds (i.e. the mean of the non-zero values across the quadrants) for each of the 3 radii, you'd request:
```
-column 'AAL_WIND_34-BAL_WIND_34,ABS(AAL_WIND_34-BAL_WIND_34),AAL_WIND_50-BAL_WIND_50,ABS(AAL_WIND_50-BAL_WIND_50),AAL_WIND_64-BAL_WIND_64,ABS(AAL_WIND_64-BAL_WIND_64)'
```
Note that these options can be specified in one long comma-separated list, or broken out with multiple calls to `-column`. Both produce the same result.

You can see why we provided some shortcuts to avoid having to write all this out! FYI, you can see where these shortcuts are defined in the code on [this line](https://github.com/dtcenter/MET/blob/d9f3ad3e2a7bdfd6c65aa612fb5e4087ac7a5171/src/libcode/vx_tc_util/tc_columns.h#L86) of `tc_columns.h`.

**Reply by JohnHalleyGotway** [reply]
@OliviaOstwald-NOAA I wanted to clarify that the full circle (i.e. "ALL") winds option being computed as the mean of the non-zero quadrants was added for MET version 11.1.0 as described in issue dtcenter/MET#2532. It's not in the "current" MET version 11.0.2.

**Comment by OliviaOstwald-NOAA** [comment]
Hi John,
Thanks so much for your response! I will give that a try. Also, I am using MET 11.1.0. I will let you know if writing out the full line gives me the results needed.

**Comment by OliviaOstwald-NOAA** [comment]
Hi John, I was able to get this to work by doing each column individually. For some reason it didn't work when doing all arguments in single quotes. Thanks so much for your help!
